src/CMake/FindZlib.cmake

```diff
@@ -58,42 +58,35 @@
 
 # Use the ZLIB_DIR hint from the config-site .cmake file 
 
-INCLUDE(${VISIT_SOURCE_DIR}/CMake/SetUpThirdParty.cmake)
+include(${VISIT_SOURCE_DIR}/CMake/SetUpThirdParty.cmake)
```

Comment:
Just curious but do we need to *find* zlib at all now that we are always, by fiat, compiling an installing it ourselves?

Reply:
SetUpThirdParty.cmake also sets us up for the installs, so yes, its needed.